lib/schema/facter.yaml

```diff
@@ -515,6 +515,20 @@ kernelversion:
         POSIX platforms: use the `uname` function to retrieve the kernel's version.
         Windows: use the file version of `kernel32.dll` to retrieve the kernel's version.
 
+ldom:
+    type: map
+    description: Return Solaris LDom information from the `virtinfo` utility.
+    resolution: |
+        Solaris: use the `virtinfo` utility to retrieve LDom information.
+    validate: false
+
+ldom_<name>:
+    pattern: ldom_\w+$
```

Comment:
Are these expected to be hidden? Normally we hide flat facts if a structured version is available.